pkg/new-ui/v1beta1/frontend/src/app/services/experiment-form.service.ts

```diff
@@ -42,6 +45,7 @@ export class ExperimentFormService {
       parallelTrialCount: 3,
       maxTrialCount: 12,
       maxFailedTrialCount: 3,
+      resumePolicy: 'Never',
```

Comment:
I think the default `resumePolicy` is `LongRunning`.

Reply:
Αhh indeed, I missed it. Thanks!

Reply:
Pushed a commit for this